src/services/fileStorage/proxy-service.js

```diff
@@ -61,6 +95,7 @@ const fileStorageService = {
 			}
 			return perm;
 		};
+		const strategy = createCorrectStrategy('awsS3');
```

Comment:
The `fileStorageType` should be in the payload set by the hooks.

Reply:
fixed